docs/data-sources/local_bower_repository.md

```diff
@@ -0,0 +1,23 @@
+---
+subcategory: "Local Repositories"
+---
+
+# Artifactory Local Bower Repository Data Source
+
+Retrieves a local Bower repository.
+
+## Example Usage
+
+```hcl
+data "artifactory_local_bower_repository" "terraform-local-test-bower-repo" {
```

Comment:
`terraform`? 😄

Reply:
Oh no its copy pasted down the list. Will remove